meson.build

```diff
@@ -0,0 +1,12 @@
+project('ttf-parser', 'rust')
+
+add_project_arguments(['--edition=2018'], language: 'rust')
```

Comment:
Does this necessary? Doesn't meson support Cargo.toml?

Reply:
It doesn't officially https://github.com/mesonbuild/meson/issues/2173

Reply:
Heated discussion here https://github.com/mesonbuild/meson/pull/2617#issuecomment-379417837 also

Reply:
I see, thanks.